web/cobrands/bristol/assets.js

```diff
@@ -45,11 +45,42 @@ fixmystreet.assets.add($.extend({}, options, {
     filter_value: 'GRITBIN'
 }));
 
-fixmystreet.assets.add($.extend({}, options, {
-    asset_category: "Street Lighting",
+var lighting_options = $.extend({}, options, {
+    wfs_url: "https://maps.bristol.gov.uk/arcgis/services/ext/datagov/MapServer/WFSServer",
```

Comment:
Given it's overriding most of the attributes of `options`, perhaps having a base options that both options and lighting_options can derive from might be a little clearer, but not much in it.